### PR TITLE
Added in the semantic properties docs

### PR DESCRIPTION
**Pull request by bijington**
Adds in docs for https://github.com/CommunityToolkit/Maui.Markup/pull/103

### CODE REVIEW

File: docs/maui/markup/extensions/semantic-properties.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+The following example demonstrates how to set the description on a new `Entry`:
+
+```cs
+new Entry().SemanticDescription("Enter your username.");
+```
+
+## SemanticHeadingLevel
+
+The `SemanticDescription` method sets a heading level to enable the`BindableObject` to be marked as a heading to organize the UI and make it easier to navigate.
```
Comment (brminnick) [inline docs/maui/markup/extensions/semantic-properties.md:30]: Is this specific to the screen reader? If so, let's add that to the docs.

I'm not sure, but `SemanticHeadingLevel` may also work for tabbing through a list of `Entry`s. If that's the case, we could add some verbiage about that too.

```suggestion
The `SemanticDescription` method sets a heading level to enable the `BindableObject` to be marked as a heading to organize the UI and make it easier to navigate for the platform's screen reader.
```

**Review by brminnick — CHANGES_REQUESTED** [review]
Looking good! I just a have a couple small suggestions.

File: docs/maui/markup/extensions/automation-properties.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+The following example demonstrates how to exclude a new `VerticalStackLayout` and its children from the accessibility tree:
+
+```cs
+new VerticalStackLayout().AutomationExcludedWithChildren(true);
+```
+
+## AutomationIsInAccessibleTree
+
+The `AutomationIsInAccessibleTree` method Sets a value determining if the `BindableObject` is visible to screen readers.
```
[the remark this answers is not in the document]
Comment (bijington) [reply docs/maui/markup/extensions/automation-properties.md:30]: Good catch!

File: docs/maui/markup/extensions/semantic-properties.md (hunk 1 above)
Comment (bijington) [reply docs/maui/markup/extensions/semantic-properties.md:30]: Yeah I like that!

File: docs/maui/markup/extensions/visual-element-extensions.md [hunk 3]
```diff
@@ -98,6 +98,16 @@ The follow example will create an `Entry` and add a [`MaxLengthReachedBehavior`]
 new Entry().Behaviors(new MaxLengthReachedBehavior());
 ```
 
+## AutomationId
+
+The `AutomationId` method sets the `AutomationId` property for the supplied `VisualElement`.
+
+The follow example will create an `Entry` and set the `AutomationId` to "PasswordEntry":
```
Comment (brminnick) [inline docs/maui/markup/extensions/visual-element-extensions.md:105]:
```suggestion
The following example will create an `Entry` and set the `AutomationId` to "PasswordEntry":
```

**Review by brminnick — CHANGES_REQUESTED** [review]
Found one more typo

**Review by jfversluis — APPROVED** [review]

**Pull request merged**